server/graphql_schemas/event/schema.py

```diff
@@ -239,9 +240,10 @@ def mutate_and_get_payload(cls, root, info, **input):
                     action_message="Event Update is successful.",
                     updated_event=updated_event
                 )
-        except ValueError as e:
+        except Exception as e:
             # return an error if something wrong happens
-            raise GraphQLError("An Error occurred. \n{}".format(e))
+            logging.warn(e)
+            raise GraphQLError("An Error occurred")
```

Comment:
Let's make this consistent
```suggestion
            raise GraphQLError("An Error occurred. Please try again")
```

Reply:
nice observation. I'll work on that